dataset/dictionary/characters.json5

```
@@ -4889,30 +4900,93 @@
     en: "Huni",
     ja: "フニ",
     zhCN: "胡妮",
-    notes: "部族見聞「ユバンキの廻焔」に登場する人物",
+    notes: "部族見聞「ユパンキの廻焔」に登場する人物",
     tags: [ "natlan", "character-sub" ],
   },
   {
     en: "Toba",
     ja: "トバ",
     zhCN: "トバ",
-    notes: "部族見聞「ユバンキの廻焔」に登場する人物",
+    notes: "部族見聞「ユパンキの廻焔」に登場する人物",
     tags: [ "natlan", "character-sub" ],
   },
   {
     en: "Sanka",
     ja: "山下",
     zhCN: "山下",
     pronunciationJa: "やました",
-    notes: "部族見聞「ユバンキの廻焔」に登場するアビス教団の使者。「淵上」と称するものと同一人物",
+    notes: "部族見聞「ユパンキの廻焔」に登場するアビス教団の使者。「淵上」と称するものと同一人物",
     notesEn: "The same chatacter as Enjou. His Japanese name is pronunced Yamashita and the English name Sanka may be a mistranslation.",
     tags: [ "natlan", "character-sub" ],
   },
   {
     en: "Trinidad",
     ja: "トリニダード",
     zhCN: "特立尼达",
-    notes: "部族見聞「ユバンキの廻焔」第二幕に登場する人物",
+    notes: "部族見聞「ユパンキの廻焔」に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Thiago",
+    ja: "ティアゴ",
+    zhCN: "蒂亚戈",
+    notes: "部族見聞「ユパンキの廻焔」第三幕に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Ponche",
+    ja: "ポンセ",
+    zhCN: "庞塞",
+    notes: "部族見聞「ユパンキの廻焔」第三幕に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Burkina",
+    ja: "ブルキナ",
+    zhCN: "布基纳",
+    notes: "部族見聞「ユパンキの廻焔」に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Kongamato",
+    ja: "コンガマトー",
+    zhCN: "康加玛托",
+    notes: "部族見聞「ユパンキの廻焔」に登場する巨大なユムカ竜",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Mountain King",
+    ja: "山の王",
+    zhCN: "山王",
+    notes: "コンガマトーへの敬称",
+    tags: [ "natlan", "character-sub", "how-to-call" ],
```

I guess Kongamato is a field boss, so `enemy-boss` tag can be added.

```suggestion
    tags: [ "natlan", "character-sub", "how-to-call", "enemy-boss" ],
```